src/main/features/core/tray.js

```diff
@@ -18,9 +18,11 @@ if (process.platform === 'darwin') {
 } else if (WindowManager.getWindowManagerGDMName() === 'kde-plasma') {
   // TODO: Change this back to ico when electron supports it on linux
   // appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main.ico`));
-  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray.png`));
+  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray_white.png`));
+} else if (process.platform === 'linux') {
+	appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray_white.png`));
 } else {
-  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray.png`));
+  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray_white.png`));
```

Comment:
Sorry, when I said the white version, I meant the `non-transparent` version with the inner being white.  I want it to stay the same on Windows.